frontend/libs/studio-components/src/components/StudioCombobox/StudioCombobox.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { StudioCombobox, type StudioComboboxProps } from './index';
+import userEvent from '@testing-library/user-event';
+
+const options = {
+  ole: 'Ole',
+  dole: 'Dole',
+  doffen: 'Doffen',
+  invalid: 'Kinkliane Koff',
+};
+const noResults = 'No results';
+
+describe('StudioCombobox', () => {
+  const verifyOnValueChange = async (
+    onValueChange: jest.Mock,
+    expectedCalls: number,
+    expectedValue: string[],
+  ) => {
+    await waitFor(() => expect(onValueChange).toHaveBeenCalledTimes(expectedCalls));
+    expect(onValueChange).toHaveBeenCalledWith(expectedValue);
+  };
+
+  it('should render the component', () => {
+    renderTestCombobox();
+    const studioCombobox = screen.getByRole('combobox');
+    expect(studioCombobox).toBeInTheDocument();
+  });
+
+  it('should render the empty state when no results are found', async () => {
+    const user = userEvent.setup();
+    renderTestCombobox();
+    const studioCombobox = screen.getByRole('combobox');
+    await user.type(studioCombobox, options.invalid);
+    const emptyState = screen.getByText(noResults);
+    expect(emptyState).toBeInTheDocument();
+  });
+
+  it('should be possible to select an option', async () => {
+    const user = userEvent.setup();
+    const onValueChange = jest.fn();
+    renderTestCombobox({ onValueChange });
+    const studioCombobox = screen.getByRole('combobox');
+    expect(studioCombobox).toHaveValue('');
+
+    await user.click(studioCombobox);
+    await user.click(screen.getByRole('option', { name: options.ole }));
+    await verifyOnValueChange(onValueChange, 1, [options.ole]);
```

Comment:
What about using an object as an argument for the `verifyOnValueChange` method? This would increase the readability where it's used. 😊 

```suggestion
    await verifyOnValueChange({
        onChange: onValueChangeMock,
        expectedNumberOfCalls: 1,
        expectedValue: [options.ole]
    });
```

Reply:
Yeah good idea :) Should I keep it as a thumb of rule that if its 3 or more, I use an object as an argument?

Reply:
I don't see the problem by following this pattern with two arguments either.  The most important is to make the API for the methods readable. F.ex `canDriveMoped(user, false)` is not that readable, as `canDriveMoped({ user, hasMotorcycleLicense: false})`. Since you can drive moped if you have motorcycle certificate, since that is "bigger" license than moped. 😊

Reply:
Got it, thx :)